tls/s2n_handshake_io.c

```diff
@@ -501,7 +517,7 @@ static message_type_t tls13_handshakes[S2N_HANDSHAKES_COUNT][S2N_MAX_HANDSHAKE_L
 };
 /* clang-format on */
 
-#define MAX_HANDSHAKE_TYPE_LEN 152
+#define MAX_HANDSHAKE_TYPE_LEN 168
```

Comment:
Is each type length 16 bytes? Should this read `MAX_HANDSHAKES * 16` or something similar?

Reply:
Unfortunately not. This is used to print the handshake type in human-readable format, like "NEGOTIATED|FULL|MIDDLEBOX_COMPAT". So the size depends on the string length of the enums involved. There's a unit test that enforces it's set right: https://github.com/aws/s2n-tls/blob/main/tests/unit/s2n_tls13_handshake_state_machine_test.c#L708-L716